_posts/0000-01-05-samizreal.md

```diff
@@ -7,3 +7,4 @@ title: "Welcome to our slide deck!"
 {: .center}
 
 Caption here
```

Comment:
Maybe you should cleanup the "Caption here"

Reply:
Good catch, Thanks!